index.js

```diff
@@ -0,0 +1,140 @@
+    mkey++
+    event.stopPropagation();
+    
+    let distIncrement = 10;
+    mvRinterval = setInterval(function () {
+        
+       
+        console.log("Inside: " + bcontainer.style.width);
+        if (distIncrement < 1785) {
+            distIncrement++;
+            event.target.style.marginLeft = distIncrement + 'px';
+        } else {
+            // event.target.style.alignSelf = "flex-end"
+            // clearInterval(mvRinterval);
+            clearInterval(orderMouse)
+            event.stopPropagation();
+            // return;
+        }
+        
+    }, 1);
+}
+function rTerminate() {
+    clearInterval(mvRinterval);
+    // clearInterval(mvLinterval)
+}
+function lTerminate() {
+    // clearInterval(mvRinterval);
+    clearInterval(mvLinterval)
+}
+
+    // event.stopPropagation();
+    // let index = 0;
+    // let ival = event.target.style.marginLeft;
+    // for (let i = 0; 92 >= i; i++) {
+    //     let perc = i + "%"
+    //     // event.target.style.alignSelf = "flex-end";
+    //     event.target.style.marginLeft = perc.toString();
+    //     if (event.target.style.paddingRight === "30px") {
+    //         event.target.style.alignSelf = "flex-end";
+
+    let mvLinterval;
+    const orderReturn = (event) => {
+        mkey++
+        event.stopPropagation();
+        // event.stopPropagation();
+        // event.target.style.alignSelf = "flex-start";
+        // event.target.style.marginRight = "1785px"
+        let distIncrement = 1785;
+        mvLinterval = setInterval(function() {
+            
+            clearInterval(orderMouse);
+            if (distIncrement >= 10) {
+                distIncrement--;
+                event.target.style.marginLeft = distIncrement + 'px';
+            }
+            else {
+                // event.target.style.alignSelf = "flex-start";
+                // clearInterval(mvLinterval);
+                clearInterval(orderReturn)
+                event.stopPropagation();
+                // return;
+            }
+        }, 1);
+
+
+
+        // for (let i = 0; 92 >= i; i++) {
+        //     let perc = i + "%"
+        //     // event.target.style.alignSelf = "flex-end";
+        //     event.target.style.marginRight = perc.toString();
+        //     if (event.target.style.paddingLeft === "30px") {
+        //         event.target.style.alignSelf = "flex-start";
+
+        //     }
+        // }
+        // event.target.style.backgroundColor = "white";
+    }
+    // const blocks = document.querySelector(".blocks")
+    // for (let i = 0; i < blocks.children.length; i++) {
+    //     blocks.children[i].addEventListener('click', orderMouse);
+    // }
+rblock.addEventListener('mousedown', orderMouse)
+bblock.addEventListener('mousedown', orderMouse)
+gblock.addEventListener('mousedown', orderMouse)
+pblock.addEventListener('mousedown', orderMouse)
+gyblock.addEventListener('mousedown', orderMouse)
+
+rblock.addEventListener('mouseup', orderReturn)
```

Comment:
when you have a lot of redundant bits of code (like 6 or so event listeners) consider wrapping them in an iterator to make it less redundant and easier to read